SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -143,6 +143,7 @@
   @JsonProperty
   private String s3UploaderBucket;
 
+  @NotEmpty
```

Comment:
`@NotEmpty` and `@NotNull` are unnecessary for `boolean` -- can't be set to `null` since it's a primitive type